pm_icecon/_types.py

```diff
@@ -1,6 +1,6 @@
-from typing import Literal, Union
+from typing import Literal
 
-Hemisphere = Union[Literal['north'], Literal['south'], str]
+Hemisphere = Literal['north', 'south']
```

Comment:
Hmmm.  I had needed to add the `str` condition when calling Hemisphere using a variable in a function.  I wonder if that still works when the `str` is removed?

Reply:
All of the tests are passing so I think my changes resolved the issue. The way we are handling hemispheres should be changing very soon though.